packages/cardpay-sdk/sdk/revenue-pool/base.ts

```diff
@@ -97,22 +97,36 @@ export default class RevenuePool {
     return gasInToken;
   }
 
+  async claim(txnHash: string): Promise<TransactionReceipt>;
   async claim(
     merchantSafeAddress: string,
     tokenAddress: string,
     amount: string,
     txnOptions?: TransactionOptions,
     contractOptions?: ContractOptions
+  ): Promise<TransactionReceipt>;
+  async claim(
+    merchantSafeAddressOrTxnHash: string,
+    tokenAddress?: string,
+    amount?: string,
+    txnOptions?: TransactionOptions,
+    contractOptions?: ContractOptions
   ): Promise<TransactionReceipt> {
+    if (!tokenAddress) {
```

Comment:
ditto with the comment about checking whether merchantSafeAddressOrTxnHash actually looks like what you expect it to be than to test a side effect.